.github/workflows/release-integration.yml

```diff
@@ -12,21 +15,6 @@ on:
         required: true
 
 jobs:
```

Comment:
This is a great! For a next step I think we could put other jobs here to run in parallel with the node-integration like sending the node PR and updating the docs. 

Then once the node-integration suite is dialed in we can make the node PR job depend on that and include a link to the workflow run in the PR body.